goentri.com.sprwt.json

```diff
@@ -25,8 +25,8 @@
     },
     {
       "type": "TXT",
-      "host": "%hostValue%",
-      "data": "%customValue%",
+      "host": "@",
+      "data": "v=spf1 include:amazonses.com include:_spf.google.com ~all",
```

Comment:
Why not SPFM???

Reply:
I tried to use SPFM but there was an error, this was the reason I used the TXT record.